rules/eslint/style/on.js

```diff
@@ -117,29 +117,29 @@ module.exports = {
     // disallow dangling underscores in identifiers
     "no-underscore-dangle": "off",
     // disallow the use of Boolean literals in conditional expressions
-    "no-unneeded-ternary": "off",
+    "no-unneeded-ternary": "error",
     // disallow whitespace before properties
-    "no-whitespace-before-property": "off",
+    "no-whitespace-before-property": "error",
     // enforce the location of single-line statements
-    "nonblock-statement-body-position": "off",
+    "nonblock-statement-body-position": ["error", "below"],
     // enforce consistent line breaks inside braces
-    "object-curly-newline": "off",
+    "object-curly-newline": ["error", { "minProperties": 2 }],
```

Comment:
I like it because it makes properties scannable. Not only that but it is consistent with how we do PropTypes though it is not consistent with how we do HTML attributes. Happy to turn it off, no strong feelings on it but that was my intention.